test.py

```diff
@@ -81,46 +140,15 @@ def target():
   return 1 if gotTimeout else process[0].returncode
 
 try:
-  subprocess.check_output(["./testmodel.py", "--help"], stderr=subprocess.STDOUT)
+  if isWin:
+    subprocess.check_output(["python", "testmodel.py", "--help"], stderr=subprocess.STDOUT)
+  else:
+    subprocess.check_output(["./testmodel.py", "--help"], stderr=subprocess.STDOUT)
```

Comment:
If you would also invoke with `python` on Linux, you could avoid this if clause entirely, and it should work just the same?

Reply:
If I remember right, the point here was also the "./" that in any case should be changed (maybe it's enough to use `normpath `also here). I'll test this.

Reply:
AFAIK, you need the `./` prefix if you want to _directly run_ an executable script/binary in the current folder. If you call `python testmodel.py`, there is no need for the `./`, so also no need for `normpath` I think.